packages/mobile/src/send/TransferConfirmationCard.tsx

```diff
@@ -67,11 +67,12 @@ const renderTopSection = (props: Props) => {
 const renderAmountSection = (props: Props) => {
   const { currency, type, value } = props
 
+  // tslint:disable react-hooks-nesting
```

Comment:
false positive because this function is called inside another render function and that lint rules can't figure it out.